frontend/package.json

```diff
@@ -81,13 +86,15 @@
     "apollo-link-error": "^1.1.12",
     "apollo-link-http": "^1.5.16",
     "bootstrap": "^4.3.1",
-    "graphql": "^14.5.8",
-    "graphql-tag": "^2.10.1",
+    "graphql": "^14.6.0",
+    "graphql-tag": "^2.10.3",
     "i18n-iso-countries": "^4.3.1",
     "lodash": "^4.17.15",
     "query-string": "^6.9.0",
-    "react-bootstrap": "^1.0.0-beta.14",
+    "react-bootstrap": "^1.0.2",
```

Comment:
This doesn't match any known versions. Running `yarn install` gives the following output:
```
warning Lockfile has incorrect entry for "react-bootstrap@^1.0.2". Ignoring it.
Couldn't find any versions for "react-bootstrap" that matches "^1.0.2"
? Please choose a version of "react-bootstrap" from this list: 
```
I chose 1.0.1 because that was what was in the lockfile. I was unable to run `yarn install --frozen-lockfile`. I think this needs to be changed to `1.0.1`.

Reply:
Not sure how that happened, fixed.